stable/redis-ha/values.yaml

```diff
@@ -79,8 +79,8 @@ redis:
   config:
     ## Additional redis conf options can be added below
     ## For all available options see http://download.redis.io/redis-stable/redis.conf
-    min-slaves-to-write: 1
-    min-slaves-max-lag: 5   # Value in seconds
+    min-replicas-to-write: 1
```

Comment:
In my experience, Redis in 5.x will automatically update these in the config to the `replicas` (I left it as a legacy compatibility) so it's strange that it's causing issues for you. But either way if it solves the problem then lets get it in. 

Can you add a small note in the README.md to explain that any Redis version older than 5.x will require these values changed to `min-slaves-to-write` and `min-slaves-max-lag`? That and simply rebase off the latest master branch since there have been many PRs getting merged in recently.